packages/node_modules/@ciscospark/spark-core/src/interceptors/redirect.js

```diff
@@ -60,6 +61,18 @@ export default class RedirectInterceptor extends Interceptor {
 
       return this.spark.request(options);
     }
+    else if (response.headers && response.body && response.body.errorCode === this.spark.config.LOCUS_REDIRECT_ERROR && response.body.location && response.options && response.options.qs && response.options.qs.alternateRedirect) {
```

Comment:
style nitpick, can you break this into multiple lines to make it easier to parse at a glance?